src/components/content/systemStatus/healthCheckStatusQuery.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ */
+
+import { useQuery } from '@tanstack/react-query';
+import { AdminService } from '../../../xpanse-api/generated';
+
+export function healthCheckStatusQuery() {
+    // eslint-disable-next-line react-hooks/rules-of-hooks
```

Comment:
@Alice1319 this comment is valid. Custom hooks must also start with the word `use`